easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCC-8.3.0.eb

```diff
@@ -0,0 +1,35 @@
+# easybuild easyconfig
```

Comment:
@fizwit this easyconfig should also be removed

Reply:
removing Brotli-1.0.9-GCC-8.3.0.eb